src/api/userInformation.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { url, headers } from '../shared/variable.shared';
+
+
+
+const getInformation = (user: any) =>{
+    return {
+        email: user.email,
+        password: "",
+        provider:"google"
```

Comment:
to chyba powinno byc brane jakos dynamicznie

Reply:
Teoretycznie tak, problem teraz jest taki, że to należy do poprzedniego brancha, 'postUserInformation', który został już wrzucony na maina. Myślę, że dopóki mamy jeden sposób logowania, może na razie zostać, ale później trzeba będzie to poprawić.